### ISSUE TITLE
Малые работы перед запуском проекта

### ISSUE THREAD

**Issue by vladimir075** [nicothin/akinita#8]
@nicothin Николай, добрый вечер. если не сложно, есть следующие просьбы по верстке. По возможности желательно, чтобы они не требовали больших переделок в проекте, который у @greeceruit уже готов к запуску. Я правда в этом не разбираюсь, поэтому наверное Олег лучше объяснит, что значит, чтобы "не требовали особых переделок в проекте".
1) нужен футер для страницы результатов поиска [ ](https://akinitamarket.gr/el/pwliseis-katoikies/ellada/)
Располагаться он может либо на всю ширину сайта как например здесь [  ](https://ru.airbnb.com/s/Skopje--Macedonia-FYROM/homes?place_id=ChIJpaKajKUVVBMRICCHYMKI7bI&refinement_paths%5B%5D=%2Fhomes&checkin=2023-10-06&checkout=2023-10-08&date_picker_type=calendar&adults=2&children=0&infants=0&pets=0&search_type=AUTOSUGGEST&fbclid=IwAR2DjbpDU0filq_MihIPNhEP0Zse0od7_x6ZX7QmZdZZ-VgYJJ-i_R4TgSw)
или на часть сайта под преьюшками, как например здесь [ ](https://www.trulia.com/LA/New_Orleans/)
Не знаю как лучше. В принципе хорошо, чтобы карта не уходила, но тогда надо будет урезать часть контента футера
2) Сверстать страницу Contact us по примеру аттачмента, только с добавлением телефона в форму. Все поля обязательны.
<img width="1594" alt="Screenshot 2023-08-07 at 3 32 23 PM" src="https://github.com/nicothin/akinita/assets/17883276/78c1ef87-82cd-478b-bdaa-1904e6cfaab8">
3) в выпадашке фильтров слово commercial в греческом варианте не помещается. можем чуть расширить выпадашку?
<img width="1031" alt="Screenshot 2023-08-07 at 9 52 58 PM" src="https://github.com/nicothin/akinita/assets/17883276/00b65d94-5c5a-48a3-9c35-777612a9d8c9">
4) в мобильнике логотип кажется немного смещен от 
<img width="382" alt="Screenshot 2023-08-07 at 9 55 45 PM" src="https://github.com/nicothin/akinita/assets/17883276/9e04224f-f0ef-4b31-a519-507577894efa">
центра 
5) Можно ли нижние прибитые кнопки как то захайлайтить? 
<img width="429" alt="Screenshot 2023-08-08 at 6 39 48 PM" src="https://github.com/nicothin/akinita/assets/17883276/614039d0-3513-4a94-9715-b42cb9c8b2ef">
например сделав их чуть серыми, как на этом примере
<img width="185" alt="Screenshot 2023-08-08 at 4 05 17 PM" src="https://github.com/nicothin/akinita/assets/17883276/6371a55b-36e4-4bc6-905d-ff6955a60235">

**Comment by nicothin** [comment]
1. добавлю такой же, как на главной стр.
2. ок, но капчу там прикручивать лучше Олегу, она, насколько помню, доменозависимая
3. сделаю шире
4. ощущение смещения, т.к. слева иконка-бургер, открывающая меню, можно сместить логотип, чтобы не было этого ощущения, сделаю.
5. именно цветом с добавлением фона - думаю, да. Но можно начать с того, что попробовать дать им более темный бордер, (благо, это быстро), и только потом двигаться в сторону фона и смены их цвета, т.к. это будет заметно "поедать" пространство, которого на мобилках совсем мало.

**Comment by vladimir075** [comment]
Ok. 
может лучше без капчи?

**Comment by greeceruit** [comment]
> Ok. может лучше без капчи?

Да. Если будут спамить, то прикрутим впоследствии.

**Comment by nicothin** [comment]
1. добавил: https://nicothin.pro/akinita/search.html
2. добавил: https://nicothin.pro/akinita/contact-us.html
3. было 120, сделал 160
4. это потому, что отсутствует блок авторизованного юзверя справа. Я дописал на блок с классом .logo временный модификатор .logo--without-user-blocks, который нужно убрать, когда блок юзверя, таки, появится.
5. для начала, сделал боле контрастный (темный) бордер

Все изменения видны в этом коммите в ветку, из которой мы смотрим на результат сборки: https://github.com/nicothin/akinita/commit/7c9f7eb3fe964b440cdd1a8297a59ebc8b7074f2

Собственно, небольшие изменения главногостилевого файла, да изменения шапки (закомментил блок пользователя, добавил класс на лого)

**Comment by vladimir075** [comment]
Николай, может забыл это написать, в контакт нужно еще поле телефона. И все поля обязательные

**Comment by nicothin** [comment]
> Николай, может забыл это написать, в контакт нужно еще поле телефона. И все поля обязательные

Нет, это я забыл поле внедрить. Там конструктор, можно любые поля добавлять, внес это изменене, ссылка та же.
Обязательность полей вписана браузерная (браузер не даст отправить форму, пока поля не заполнены корректно, совместимость - https://caniuse.com/mdn-api_htmlinputelement_required). Нужна какая-то другая?

**Comment by greeceruit** [comment]
@nicothin , можете добавить в sprite-svg.svg иконку телеграма? Хотим её поставить вместо твиттера в футер
![Screenshot from 2023-08-28 21-26-31](https://github.com/nicothin/akinita/assets/29063417/27e446ce-e020-49a0-a710-f7c625634315)

**Comment by nicothin** [comment]
добавил в спрайт и разместил на месте твиттера в подвалах: https://github.com/nicothin/akinita/commit/36f809aa8fcc904ca2d9a85364de09e126eaae21

**Comment by greeceruit** [comment]
@nicothin Ещё одна небольшая правка.... На странице результатов поиска https://nicothin.pro/akinita/search.html  нужно уменьшить пространство между знаком евро и ценой (т.е. сделать, как будто там нет пробела ). Тем где .card__price-currency {  margin-right: 0.5em; }

**Comment by nicothin** [comment]
> @nicothin Ещё одна небольшая правка.... На странице результатов поиска https://nicothin.pro/akinita/search.html нужно уменьшить пространство между знаком евро и ценой (т.е. сделать, как будто там нет пробела ). Тем где .card__price-currency { margin-right: 0.5em; }

https://github.com/nicothin/akinita/commit/7099f62230881de81ee2779f9debef256f58d41e - коммит в соотв. ветку

**Comment by vladimir075** [comment]
@nicothin Николай, а можно сделать превьюшки цен на карте адаптивные по размеру значения цены? 
![Uploading Screenshot 2023-09-13 at 2.41.13 PM.png…]()

**Comment by vladimir075** [comment]
Николай, а можно сделать превьюшки цен на карте адаптивные по размеру значения цены?
<img width="594" alt="Screenshot 2023-09-13 at 2 41 13 PM" src="https://github.com/nicothin/akinita/assets/17883276/2b4b81bd-7616-4bef-ad8b-55d7bdabfc78">
<img width="650" alt="Screenshot 2023-09-13 at 2 41 20 PM" src="https://github.com/nicothin/akinita/assets/17883276/b1bd7b5b-d965-4aa2-9afa-baafcfe920d1">

**Comment by greeceruit** [comment]
> Николай, а можно сделать превьюшки цен на карте адаптивные по размеру значения цены? 

@vladimir075  @nicothin  У Николая есть эти плашки трёх разных размеров. https://nicothin.pro/akinita/search.html Я внедрю

**Comment by nicothin** [comment]
да, там, кажется, абс. позиционирование было (проблемы с резиновостью плашки), но чтоб точно сказать, мне нужна ссылка на страницу с картой и плашками...

**Comment by greeceruit** [comment]
> да, там, кажется, абс. позиционирование было (проблемы с резиновостью плашки), но чтоб точно сказать, мне нужна ссылка на страницу с картой и плашками...

https://akinitamarket.gr/enoikiaseis-katoikies/ellada/
https://akinitamarket.gr/epoxiakes-enoikiaseis-katoikies/ellada/
https://akinitamarket.gr/pwliseis-katoikies/ellada/timi-apo-700000/

**Comment by vladimir075** [comment]
Олег вроде поставил там разные размеры, но в идеале конечно, если это возможно, сделать ее и вправду резиновой. Потому что даже на самом маленьком размере, все равно там лишнее место остается

**Comment by nicothin** [comment]
> Олег вроде поставил там разные размеры, но в идеале конечно, если это возможно, сделать ее и вправду резиновой. Потому что даже на самом маленьком размере, все равно там лишнее место остается

думаю, это можно попробовать сделать, в выходные попробую.
